### PR TITLE
PR template: fix both checklist items reported in #855

### PR DESCRIPTION
**Pull request by crcrewso**
# Pull Request Template

## Summary

Reported by @leducvin, who hit the second one for real on PR #825: believing there were no pending migrations, because the command printed nothing.

Updates template to clarify what each action does and fixes a clear problem with checking if migrations are introduced within a PR.  

## Related Issues

Closes #855 

## Type of Change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature ( adds functionality, always assumed to be breaking until tested by maintainers)
- [ ] Breaking change (fix or addition that changes existing behaviour)
- [x] Documentation only

## Target Branch

_Confirm which branch this targets (e.g. master)._

## AI Usage Disclosure

- [ ] No AI tools were used in this PR.
- [x] AI tools were used:
  - **Tools / where used:** _e.g. GitHub Copilot for boilerplate_
  - [x] I've reviewed every AI-generated line as if I wrote it myself, and confirm: no PHI shared, no license-incompatible content (Apache-2.0).

---

### Coder Tasks

- [x] Rebased / merged `develop` branch; no merge conflicts.
- [x] `uv run pre-commit run --all-files` passes (ruff lint, ruff-format, django-upgrade).
  - `uv run ruff check . --fix` to help resolve linting errors
- [x] `uv run python -Wd manage.py check` reviewed for new deprecation warnings.
- [x] `uv run pytest` full test suite passes.
- [x] `uv run python manage.py makemigrations --check --dry-run` if migrations are found, change should be marked Breaking
- [ ] Docker builds succeed if touched.
- [ ] Docs updated under `docs/` if behaviour, settings, or APIs changed.
- [ ] `release_notes.md` updated if user-facing.
- [ ] No secrets, debug code, or stray `print`/`console.log` left in.

### Review Code

- [ ] Reviewer verifies the target branch.
- [ ] For non-trivial changes: pulled the branch, ran it locally, and confirmed it works as described.
- [ ] CI is green; test suite and `pre-commit` pass.
- [ ] Reads through all changed files for anything weird or unintended.
- [ ] Logic is correct; edge cases and error handling considered.
- [ ] Migrations are sane and reversible where possible.
- [ ] Changes are consistent with the modernization direction (no reintroduction
      of patterns being migrated away from, e.g. flake8/yapf-era conventions).
- [ ] No security concerns (input validation, permissions, injection, XSS).
- [ ] Comments are sufficient and not overkill.
- [ ] Typos, spellcheck, professional.
- [ ] Reviewer sends back review.

### Edits

- [ ] Coder fixes any issues.
- [ ] Reviewer approves.

> Maintainer review is required for breaking changes, new features, and migrations.
> For bug fixes, refactors, and docs-only changes, reviewer approval is sufficient to merge.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -33,11 +33,17 @@ _Confirm which branch this targets (e.g. master)._
...
-  - `uv run ruff check . --fix` to help resolve linting errors
+- [ ] `uv run pre-commit run --all-files` passes 
+- This contains ruff lint check, django-upgrade (4.2), django system check, plus the yaml/toml/large-file/merge-conflict/debug-statement checks).
+- `uv run ruff check . --fix` to help resolve linting errors
 - [ ] `uv run python -Wd manage.py check` reviewed for new deprecation warnings.
 - [ ] `uv run pytest` full test suite passes.
-- [ ] `uv run python manage.py makemigrations --check --dry-run` if migrations are found, change should be marked Breaking
+- [ ] `uv run python manage.py makemigrations --dry-run`
+- This requires a working local_settings.py and connected db, including:
+  - uv run python manage.py migrate, createcachetable, collectstatic, and createsuperuser so that makemigrations doesnt recreate existing migrations (updated fix to be published before 4.1)
```
[the remark this answers is not in the document]
Comment (crcrewso) [reply .github/pull_request_template.md]: this is a note, not a copyable command, the absence of `'s is intentional.

[the remark this answers is not in the document]
Comment (crcrewso) [reply .github/pull_request_template.md]: intentionally not exploring --frozen at this time

**Review by nsmela — APPROVED** [review]
approved
